pom.xml

```diff
@@ -21,30 +21,33 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!--Dependency Properties-->
-    <raml-module-builder.version>35.2.0</raml-module-builder.version>
-    <vertx.version>4.5.4</vertx.version>
+    <raml-module-builder.version>35.3.0</raml-module-builder.version>
+    <vertx.version>4.5.10</vertx.version>
     <postgres.image>postgres:12-alpine</postgres.image>
```

Comment:
seems we do not use this property postgres.image, in this case it can be deleted